dsgrid/registry/project_registry_manager.py

```diff
@@ -1126,7 +1126,6 @@ def _make_dimension_associations(
         df = config.make_dimension_association_table(dataset_id, context)
         path = context.get_temp_filename(suffix=".parquet")
         df = write_dataframe_and_auto_partition(df, path)
-        context.add_tracked_path(path)
```

Comment:
This seems like a non sequitur

Reply:
Kinda. `context.get_temp_filename` previously did not add tracking of the returned filename. I found it very easy to forget to add the path afterwards. So, I combined the steps into one and removed this call.